generators/server/templates/server/src/service/user.service.ts.ejs

```diff
@@ -37,6 +38,7 @@ export class UserService {
 
   async save(user: User): Promise<User | undefined> {
     user = this.convertInAuthorities(user);
+    user.password = await generateHashedPassword(user.password);
```

Comment:
Maybe you have to move this encrypt method to the user.repository class in save methods.
Then, you have to add a decrypt method in findOne repository method. The reason is that the integration tests for security and entity fail as result of not decrypt in the rest get api

Reply:
ok, I will do as corrections

Reply:
I will make corrections